### DISCUSSION TITLE
The limit of characters that can be sent through a push notification

### DISCUSSION THREAD

**Discussion by sam9010** [invertase/react-native-firebase#4592] [Q&A] [answered]
i want to show 1000 characters( or all characters )  in push notification,
how to show all them? do not show line`s first from our text(body)

**Comment by andersonaddo** [comment, marked as answer]
Your post isn't very clear so I'm not exactly sure how to help, maybe this will be helpful:
https://stackoverflow.com/questions/53870080/actual-android-push-message-character-limit
